test/Constraints/optional.swift

```diff
@@ -565,3 +565,18 @@ func testFunctionContainerMethodCall(container: FunctionContainer?) {
   // expected-note@-3 {{coalesce}}
   // expected-note@-4 {{force-unwrap}}
 }
+
+// Test for Issue: 60730 (https://github.com/apple/swift/issues/60730)
+// Rdar: rdar://94037733
+func issue60730Test() {
```

Comment:
A couple of suggestions here:
1. Let's adjust the comment so that it just mentions links, they are self-evident.
2. Don't have to add a function, `do { }` could be used instead to outline the scope.

Reply:
Done